### PR TITLE
Portfolio Website: Add Desktop Version

### PR DESCRIPTION
**Pull request by rmiyoyo**
# <span style="color:red">NOTE</span>
I erroneously merged the first [pull request](https://github.com/rmiyoyo/Portfolio-Mobile-Version-Completion/pull/3) after the first review.

# Work Done After First Review
- Reduce the spacing above the header to match the design template.
- Fixed styling issues on project cards.
- Implemented the flipping of images and their descriptions in the projects section.
- Fixed margin sizes in work section.
- Reduced padding in images.
- Removed all unnecessary spacing.

### CODE REVIEW

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @rmiyoyo  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- I think your skills/languages section is kind of squashed up. Can we fix it?? If you like, You may consider using `<divs>` to separate them instead of the `<li>` tags that you have used, this will make it easier to differentiate between the two sections. 🌸 
- Can we fix the text issue? This is what i mean 👇🏽 

Here is what I can see compared to what's on Figma 👇🏽 
| Your languages/skills section  |
| :---:   | 
|Screenshot 2023-04-01 at 09 40 24
  |


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rmiyoyo** [comment]
# Work Done After Second Review
- Fixed language/skills section by adding spacing between items.
- Ran linters checks

File: style.css [hunk 1]
```diff
@@ -533,28 +563,80 @@ body {
...
+  .skill-name {
+    position: absolute;
+    margin-left: -12%;
+    margin-top: 13%;
+    color: #253858;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 20px;
+  }
```
Comment (IngDavidTM) [inline style.css:625]: - [OPTIONAL] You have done a fantastic job with your work :+1:. However, please take a moment to check each language that you added to your languages section. Ensure that the text (JavaScript, HTML and CSS) is inside the same background as shown in the Figma template. This is crucial to maintain a consistent design and ensure that the page looks polished and professional. By matching the text background to the template, you can provide users with a more cohesive and visually appealing experience.

**Your design**

Screenshot from 2023-04-01 10-48-56

**Figma**

Screenshot from 2023-04-01 10-49-59

**Review by IngDavidTM — APPROVED** [review]
Hi @rmiyoyo  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [OPTIONAL] I kindly request that you check the height and width of each card inside the works section. It appears that the cards are bigger than expected. To address this issue, I recommend giving each card a `max-width` and `max-height`. This will ensure that the cards fit neatly within the page and do not overflow, creating a better visual experience for users. By fixing the size of the cards, you can provide users with a more visually appealing and professional-looking page.

**Your design**

Screenshot from 2023-04-01 10-50-45

**Figma**

Screenshot from 2023-04-01 10-50-21


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (rmiyoyo) [reply style.css:625]: Thank you very much @IngDavidTM  for your output. I am integrating these changes.

**Pull request merged**